coverage.xml

```diff
@@ -0,0 +1,942 @@
+<?xml version="1.0" ?>
```

Comment:
You presumably didn't mean to add this file?

Reply:
Yes, thanks!